superset/db_engine_specs/kustosql.py

```diff
@@ -0,0 +1,91 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Nit - to avoid clutter, could we place both the `KustoKqlEngineSpec` and `KustoSqlEngineSpec` in one file `superset/db_engine_specs/kusto.py`?

Reply:
We kept two separate modules in our fork for historical reasons. But I totally agree that we should merge them in a single file to keep the overall project structure cleaner. Thank you for pointing this, and I will refactor this code.